blocks/algorithms/__init__.py

```diff
@@ -276,7 +278,7 @@ def compute_steps(self, gradients):
 
 
 class SteepestDescent(StepRule):
-    """A step in the direction opposite to the gradient.
+    """A step in the direction proportional to the gradient.
```

Comment:
The whole purpose of this pull request is to ensure that the input is a gradient (clipped, accumulated), not negative gradient.

Renaming `SteepestDescent`  into `Scale` is a reasonable thing to do.